npm/pkg/controlplane/controllers/v2/networkPolicyController_test.go

```diff
@@ -302,24 +303,71 @@ func TestAddMultipleNetworkPolicies(t *testing.T) {
 
 	dp := dpmocks.NewMockGenericDataplane(ctrl)
 	f.newNetPolController(stopCh, dp, false)
+	var testCases []expectedNetPolValues
+
+	if util.IsWindowsDP() {
+		dp.EXPECT().UpdatePolicy(gomock.Any()).Times(0)
+		addNetPol(f, netPolObj1)
```

Comment:
nit: might make things slightly easier to read if we move the duplicate code out of the if-else (here and other functions)

Reply:
refactored, thanks